### ISSUE TITLE
How do I check what current_user is in a view other than the application layout view?

### ISSUE THREAD

**Issue by mklophaus** [mklophaus/farmer_stand#1]
I'm using STI and I'm trying to change a layout or two in my project by checking the current_user... I want to check whether or not the user is a "Farmer" or "Purchaser" in the view

I put this in the controller ![](https://i.imgur.com/xzUneAj.png) thinking it would allow me to check in the corresponding view for this but couldn't figure out how to get it to work....

So far I've tried <% if  current_user.type == "Farmer" %> and
<% if current_user.is_a "Farmer" %>

 but the error keeps saying this is a nil method.

Basically I'm trying to prevent creating two forms for a "Purchaser" or "Farmer". I only want to make some small changes for the "Purchaser" form 

Thanks!

Mike
